sound/soc/sof/sof-audio.h

```diff
@@ -343,6 +343,13 @@ struct snd_sof_dai_link {
 	struct list_head list;
 };
 
+/* Pin bindings for special pins of a widget */
+struct snd_sof_pin_binding {
+	int bind_type;
+	int queue_id;
+	char wname[SNDRV_CTL_ELEM_ID_NAME_MAXLEN];
+};
```

Comment:
please add comments or kernel-doc, I personally have no idea of what this represents.

The reviewer shouldn't have to make any guesswork. The code should be self-explanatory.